src/components/organisms/Table/stories/v2.stories.tsx

```diff
@@ -0,0 +1,306 @@
+  "december": 343,
+  "average": 1000
+  },
+  {
+  "id": 6,
+  "employee": "Justin Song",
+  "january": 1282,
+  "february": 43,
+  "march": 1784,
+  "april": 4,
+  "may": 1492,
+  "june": 381,
+  "july": 1422,
+  "august": 1251,
+  "september": 738,
+  "october": 274,
+  "november": 1536,
+  "december": 1975,
+  "average": 1000
+  },
+  {
+  "id": 7,
+  "employee": "Marshall Barton",
+  "january": 1635,
+  "february": 135,
+  "march": 1566,
+  "april": 922,
+  "may": 953,
+  "june": 1191,
+  "july": 1597,
+  "august": 1605,
+  "september": 1443,
+  "october": 913,
+  "november": 287,
+  "december": 1774,
+  "average": 1000
+  },
+  {
+  "id": 8,
+  "employee": "Tamara Riley",
+  "january": 1463,
+  "february": 616,
+  "march": 1722,
+  "april": 987,
+  "may": 1925,
+  "june": 65,
+  "july": 315,
+  "august": 1777,
+  "september": 1497,
+  "october": 1610,
+  "november": 1248,
+  "december": 866,
+  "average": 1000
+  }
+  ]
+
+const metaData = [
+    {
+    "id": 1,
+    "employee": "forecast",
+    "january": 1519,
+    "february": 390,
+    "march": 1126,
+    "april": 746,
+    "may": 297,
+    "june": 789,
+    "july": 721,
+    "august": 369,
+    "september": 232,
+    "october": 434,
+    "november": 489,
+    "december": 1,
+    "average": 1000
+    },
+    {
+    "id": 2,
+    "employee": "turnover",
+    "january": 1212,
+    "february": 1133,
+    "march": 1116,
+    "april": 1530,
+    "may": 1659,
+    "june": 902,
+    "july": 1321,
+    "august": 1929,
+    "september": 297,
+    "october": 1192,
+    "november": 1957,
+    "december": 1646,
+    "average": 1000
+    },
+    {
+    "id": 3,
+    "employee": "cost",
+    "january": 1057,
+    "february": 268,
+    "march": 1854,
+    "april": 234,
+    "may": 1893,
+    "june": 1311,
+    "july": 532,
+    "august": 171,
+    "september": 871,
+    "october": 627,
+    "november": 592,
+    "december": 928,
+    "average": 1000
+    },
+    {
+    "id": 4,
+    "employee": "difference",
+    "january": 1692,
+    "february": 210,
+    "march": 391,
+    "april": 944,
+    "may": 1641,
+    "june": 809,
+    "july": 202,
+    "august": 270,
+    "september": 1020,
+    "october": 180,
+    "november": 976,
+    "december": 1799,
+    "average": 1000
+    }
+]
+
+export const TableV2: React.FC = () => {
+  const [currentData, setCurrentData] = React.useState<TableRow[]>(userData as TableRow[])
+
+  return (
+    <Table sortData={currentData} originalData={userData} setSortData={setCurrentData}>
+      <Head>
+        <Row>
+          <Cell type='head' value='full name' field='employee' sortable color='success' />
+          <Cell type='head' value='january' field='january' />
+          <Cell type='head' value='february' field='february' />
+          <Cell type='head' value='march' field='march' />
+          <Cell type='head' value='april' field='april' />
+          <Cell type='head' value='may' field='may' />
+          <Cell type='head' value='june' field='june' />
```

Comment:
Why do the `Cell` component need a `field` props?

Reply:
'field' will take the key name that will be used to access the value inside the given object data.'
'value' can be custom but field has to match exactly

Reply:
Yes I know we already speek about it 😅 But for the minimal component it's better to just handle a `value` props and a `children`. Lets have another discussion about it monday.